pkg/apis/sriovnetwork/v1/sriovnetworknodepolicy_types.go

```diff
@@ -21,6 +21,9 @@ type SriovNetworkNodePolicySpec struct {
 	NicSelector  SriovNetworkNicSelector `json:"nicSelector"`
 	DeviceType   string                  `json:"deviceType,omitempty"`
 	IsRdma       bool                    `json:"isRdma,omitempty"`
+	TotalVfs     int                     `json:"totalVfs,omitempty"`
```

Comment:
The `TotalVfs` in SriovNetworkNodePolicy may looks confusing to users, since not all the NICs support configuring this parameter. And I think, from function perspective, specifying `NumVfs` is enough.